jenkins/pipelines/cd/atom-jobs/build-common.groovy

```diff
@@ -299,15 +299,15 @@ def checkoutCode() {
         """
         sh "chown -R 1000:1000 ./"
     } else {
-        def codeCacheInFileserverUrl = "${FILE_SERVER_URL}/download/cicd/daily-cache-code/src-${REPO}.tar.gz"
+        def codeCacheInFileserverUrl = "cicd/daily-cache-code/src-${REPO}.tar.gz"
         def cacheExisted = sh(returnStatus: true, script: """
-            if curl --output /dev/null --silent --head --fail ${codeCacheInFileserverUrl}; then exit 0; else exit 1; fi
+            if curl --output /dev/null --silent --head --fail ${FILE_SERVER_URL}/download/${codeCacheInFileserverUrl}; then exit 0; else exit 1; fi
```

Comment:
How about implement a function named  `isExisted` to wrap this?

Reply:
And can we judge it by ks3 tool?

Reply:
I'm going to remove this step, these code should always be cached

Reply:
fix in another PR